src/mongo/MongoScrapbook.ts

```diff
@@ -131,8 +136,24 @@ async function executeCommand(context: IActionContext, command: MongoCommand): P
                 const docNode = new MongoDocumentTreeItem(colNode, document);
                 await ext.fileSystem.showTextDocument(docNode, { viewColumn: vscode.ViewColumn.Beside });
             } else {
-                const viewColumn = vscode.window.activeTextEditor?.viewColumn;
-                await vscodeUtil.showNewFile(result, 'result', '.json', viewColumn ? viewColumn + 1 : undefined);
+                const showOptions: vscode.TextDocumentShowOptions = { viewColumn: vscode.ViewColumn.Beside };
+                const label: string = 'Scrapbook-results';
+                const fullId: string = `${database.fullId}/${label}`;
+                readOnlyContent = readOnlyContent || resultMap.get(fullId);
```

Comment:
Why do you need the `resultMap`? I thought it was only used for "Execute All", but that should be handled by passing in `readOnlyContent`

Reply:
With the new "Execute All" implementation, `resultMap` is only used for appending each time plain old `executeCommand` is called (without passing in a value for `readOnlyContent`).

But it's overkill since that map would only ever have one item in it. I'll update it